lib/plugin.js

```diff
@@ -24,6 +24,11 @@ ManifestPlugin.prototype.getFileType = function(str) {
   return ext;
 };
 
+ManifestPlugin.prototype.existNameInManifest = function(name, manifest) {
```

Comment:
Rather than checking if a value is already present in the manifest (Which I can see could conflict with #55 , just make sure that the asset is a real asset, not something generated by webpack (make sure that `asset.chunks` is empty). Not sure if I make sense?